cfme/services/service_catalogs/ui.py

```diff
@@ -1,3 +1,5 @@
+import pytest
```

Comment:
**REQUIRED** We shouldn't ever import pytest into a model. We can raise a NotImplemented error which I think is now caught and turned in to a skip